testsuite/features/core/proxy_register_as_minion_with_script.feature

```diff
@@ -22,9 +22,9 @@ Feature: Setup SUSE Manager proxy
 @proxy
   Scenario: Accept the key for the proxy
     Given I am authorized as "testing" with password "testing"
-    When I go to the minion onboarding page
-    Then I should see a "pending" text
-    When I accept "proxy" key
+    And I go to the minion onboarding page
```

Comment:
When

Reply:
Do you mean **all of them** to start with `when`, like this:
```
    Given I am authorized as "testing" with password "testing"
    When I go to the minion onboarding page
    When I wait until I see "pending" text, refreshing the page
    When I accept "proxy" key
```    
Or you want instead I change just the last `And` with `When`?

Reply:
> Do you mean **all of them** to start with `when`, like this:
> 
> ```
>     Given I am authorized as "testing" with password "testing"
>     When I go to the minion onboarding page
>     When I wait until I see "pending" text, refreshing the page
>     When I accept "proxy" key
> ```
> 
> Or you want instead I change just the last `And` with `When`?

Please change just first "And" into "When". The following "And"s stay.
```
Given
When
And
And
```